efopen/ef_generate.py

```diff
@@ -688,7 +688,10 @@ def main():
 
     # 1. CONDITIONALLY MAKE ROLE AND/OR INSTANCE PROFILE FOR THE SERVICE
     # If service gets a role, create with either a custom or default AssumeRole policy document
-    conditionally_create_role(target_name, sr_entry)
+    if 'tags' in sr_entry.keys():
+      conditionally_create_role(target_name, sr_entry, tags=sr_entry['tags'])
```

Comment:
I'd remove this entire block and simplify it down to 
```suggestion
      conditionally_create_role(target_name, sr_entry, tags=sr_entry.get('tags'))
```
Since `conditionally_create_role` accepts `None` by default, this will give it exactly what it wants.

Reply:
Done